### DISCUSSION TITLE
feat: How to use sharrefArrayBuffer?

### DISCUSSION THREAD

**Discussion by arasrezaei** [ionic-team/capacitor#3945] [Q&A]
Hi there,
I am using ffmpeg-wasm. It works great on desktop.
But It's not working on mobile because sharedarraybuffer is not active on mobile. Is there any solution to make it working? My app is not going to be published and I am aware of enabling SAB vulnerabilities.
But I could'nt enable it on android.
How to overcome this? 
I found our this feature is enabled in firefox android but chrome and safari does not have it.
I checked out the chrome flags but I did not find it.
What's the options? I used ffmpeg mobile (cordova plugin)
But it's as slow as hell, and it freezes ui, and does not support async operations.

**Comment by Philipp-Sc** [comment]
stockfish.worker.js:1 [Deprecation] SharedArrayBuffer will require cross-origin isolation as of M91, around May 2021. See https://developer.chrome.com/blog/enabling-shared-array-buffer/ for more details.

Same problem with Stockfish https://github.com/niklasf/stockfish.wasm
How to enable it?

**Comment by thomasvidas** [comment]
Unfortunately the underlying iOS and Android browsers don't have support for `SharedArrayBuffer` due to the [Meltdown and Spectre](https://meltdownattack.com/) security vulnerabilities and at this time it doesn't seem like there are plans to add in support for it in Capacitor.

Your best best is to make your own plugin using [FFmpeg-kit](https://github.com/tanersener/ffmpeg-kit)
